File: src/sdk-reference/cpp/1/collection/get-mapping/snippets/get-mapping.cpp
try {
  std::string mapping = kuzzle->collection->getMapping("nyc-open-data", "yellow-taxi");

  std::cout << mapping << std::endl;
  // {"dynamic": "false","_meta": { "area": "Panipokhari" }, "properties":{"license":{"type":"keyword"},"driver":{"properties":{"name":{"type":"keyword"},"curriculum":{"type":"text"}}}}}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/get-mapping/snippets/get-mapping.cs
try {
  string mapping = kuzzle.collection.getMapping("nyc-open-data", "yellow-taxi");

  Console.WriteLine(mapping);
  // {"properties":{"license":{"type":"keyword"},"driver":{"properties":{"name":{"type":"keyword"},"curriculum":{"type":"text"}}}}}
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}
